open-craft/edx-platform
# BB-1744: Cherry pick pymongo upgrade from upstream

This PR cherry picks changes from the upstream merged PR.

**Testing instructions:**
1. Check out branch on devstack.
2. Run `paver install_prereqs` on both LMS and Studio shell (or just `pip install pymongo==3.9.0` to save time)
3. Perform the testing checking (only the pieces related to mongo, like creating course, editing sections, adding blocks and imports/export course).

**Reviewer:**
- [x] @lgp171188 or @viadanna
